src/GraphQL.SystemTextJson/StringExtensions.cs

```diff
@@ -25,6 +25,16 @@ public static class StringExtensions
             }
         };
 
+        private static readonly JsonSerializerOptions _jsonOptions2 = new JsonSerializerOptions
```

Comment:
Why new field ?

Reply:
The other one is used by `ToDictionary`.